docs/dataset_prepare.md

```diff
@@ -163,3 +175,29 @@ python tools/convert_datasets/stare.py /path/to/stare-images.tar /path/to/labels
 ```
 
 The script will make directory structure automatically.
+
+### A2D2
+
+To set up the A2D2 semantic segmentation dataset, first download 'Dataset - Semantic Segmentation' (`camera_lidar_semantic.tar`) from the official site ([a2d2.audi/a2d2/en/download.html](https://www.a2d2.audi/a2d2/en/download.html)). Extract the downloaded file as `a2d2/camera_lidar_semantic/`. Inside this directory, there will be 10 subdirectories, each containing separate `camera`, `label`, `lidar` subfolders.
+
+Next, while in MMSegmentation directory root, create a symbolic link from `mmsegmentation/data/a2d2` to the dataset directory `a2d2/camera_lidar_semantic`.
+
+```shell
+ln -s data/a2d2/ /absolute/path/to/a2d2/camera_lidar_semantic/
```

Comment:
ln -s data/a2d2/ /absolute/path/to/a2d2/camera_lidar_semantic/ --> 
ln -s /absolute/path/to/a2d2/camera_lidar_semantic/ data/a2d2/ ?

Reply:
Correct - keen observation. Thank you!